react/src/js/react/posts/PostView.jsx

```diff
@@ -1,25 +1,87 @@
 import React from "react";
 import Reflux from "reflux";
 
-import { Alert } from "react-bootstrap";
+import { Alert, Button, Form, FormGroup, FormControl, Thumbnail } from "react-bootstrap";
+import ReactMarkdown from "react-markdown";
 
 import { PostsStore, PostsActions } from "../discover/PostsStore";
 import LoadingComponent from "../misc/LoadingComponent";
+import { formatDate } from "../util/DateUtil";
 import Post from "./Post";
+import AuthStore from "../auth/AuthStore";
+import { Link } from "react-router-dom";
 
 /**
  * Renders details about one specific post
  */
 export default class PostView extends Reflux.Component {
     constructor(props) {
         super(props);
-        this.store = PostsStore;
+        this.stores = [PostsStore, AuthStore];
     }
 
     componentDidMount() {
         PostsActions.getPost(this.props.match.params.id);
```

Comment:
You need to load more comments from the database here. The post by default only contains the 5 most recent ones. We need pagination style comment loading though. On initial mount, load the first 25 or whatever, then you need to add in a way to load more once the user scrolls to the bottom of the comments list (IF there is more)